packages/donations-app/app/components/home/homeView.html

```diff
@@ -1,200 +1,174 @@
-      <img src="https://alex-s3-hatskier.s3.amazonaws.com/1534336890389_tom-parsons-426898-unsplash.jpg"/>
-    </div>
-  </div>
+           <!-- Mobile -->
+           <div class="col-xs-12 hidden-md hidden-lg no-padding">
+             <div class="form-group col-xs-12 no-padding">
+               <button ng-click="homeCtrl.explore()">
+                 <i class="white glyphicon glyphicon-chevron-right" style="transform: translateY(1px); margin-left: -10px"></i>
+                 EXPLORE PROJECTS
+               </button>
+             </div>
+           </div>
+         </div>
+       </div>
+     </div>
+   </section>
 
-  <div id="home-projects" class="home-projects-wrapper home-section col-md-12">
-    <img id="home-projects-logo" src="https://s3.eu-west-2.amazonaws.com/alice-res/AliceLogoWhite.png"/>
-    <div class="home-projects-text">
-      <div class="home-projects-title">PROJECTS</div>
-      <div class="section-heading-border"></div>
-      <div class="home-projects-subtitle">
-        <span class="subtitle-title">PROVIDING YOU LARGE CHOICE FOR YOUR DONATIONS</span><br><br>
-        VIEW A SMALL SELECTION BELOW
-      </div>
-    </div>
+   <section class="container">
+     <div class="row">
+       <h3 class="uppercase">
+         Explore our social projects
+       </h3>
+     </div>
+     <div class="row">
+         <homepage-project-card ng-repeat="project in homeCtrl.projects"
+                                project="project"
+                                dashboard="'project'">
+         </homepage-project-card>
+     </div>
+   </section>
 
-
-    <div class="projects-carousel" uib-carousel active="activeStory" interval="10000" no-wrap="false">
-      <div uib-slide ng-repeat="slide in homeCtrl.projects track by $index" index="$index">
-        <div id="projects-carousel" class="carousel-wrapper col-md-12 col-xs-12">
-          <div class="circle-image-wrapper col-md-3 col-xs-12">
-            <div class="carousel-image-cropper">
-              <a href="/project/{{slide.projectCode}}/"><img ng-src="{{slide.projectImage}}" class="carousel-image"></a>
-            </div>
-          </div>
-          <!--TODO parametrize the defualt landing mode for home page links -->
-          <a href="/#/project/{{slide.projectCode}}/2">
-            <div class="col-md-5 col-md-offset-1 col-xs-12 carousel-text">
-              <h1 style="color: #38D2DA;">{{ slide.projectTitle }}</h1>
-              <div class="project-charity"> {{ slide.projectCharity }} </div>
-              <div ng-bind-html="slide.projectDetails"></div>
+   <section class="container">
+     <div class="row">
+       <h3 class="uppercase">
+         Social organisations we work with
+       </h3>
+     </div>
+     <div class="row home-charity-container">
+       <div class="col-md-2"
+            ng-repeat="charity in homeCtrl.charities | limitTo: 6">
+            <div class="home-charity">
+              <img src="{{charity.picture}}" class="home-charity-picture">
```

Comment:
The logos are distorted as the width/height ratio is changed (either we remove max-height or the 100% width, but you're the expert here :)